mini_pupper_description/config/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.5)
+project(mini_pupper_description)
```

Comment:
do we need these CMakelists?

1. mini_pupper_description/config/CMakeLists.txt
2. mini_pupper_description/config/champ/CMakeLists.txt
3. mini_pupper_description/meshes/CMakeLists.txt

Reply:
actually not necessary since there are no source files, just to make sure that the package is built correctly since i found some incorrect building during testing process